test/spec/modules/connatixBidAdapter_spec.js

```diff
@@ -52,7 +62,7 @@ describe('connatixBidAdapter', function () {
       delete bid.mediaTypes;
       expect(spec.isBidRequestValid(bid)).to.be.false;
     });
-    it('Should return false if banner is missing from mediaTypes ', function () {
+    it('Should return false if banner and video is missing from mediaTypes ', function () {
```

Comment:
banner OR video instead of AND

Reply:
It should return false if both `banner` and `video` are missing from `mediaTypes`. If only one of them is missing and the other one is present, the check would go through and return true. So I think _and_ is more appropriate.

Reply:
...if none of banner and video is missing....

from the current 'it' statement I understand that both banner and video should be present otherwise it returns false.